korrekt/src/test/integration_tests_zcash.rs

```diff
@@ -101,6 +101,38 @@ mod tests {
         assert!(output_status.eq(&AnalyzerOutputStatus::NotUnderconstrainedLocal));
     }
 
+    #[test]
+    fn no_selector_test() {
+        let circuit =
+            sample_circuits::simple::no_selector::MulCircuit::<Fr>::default(
+            );
+        let k: u32 = 3;
+
+        let mut analyzer = Analyzer::new(&circuit, k).unwrap();
+
+        assert!(analyzer.instace_cells.clone().len().eq(&2));
+
+        let modulus = Fr::MODULUS;
+        let without_prefix = modulus.trim_start_matches("0x");
+        let prime = BigInt::from_str_radix(without_prefix, 16)
+            .unwrap()
+            .to_string();
+
+        let analyzer_input: analyzer_io_type::AnalyzerInput = analyzer_io_type::AnalyzerInput {
+            verification_method: VerificationMethod::Random,
+            verification_input: VerificationInput {
+                instances_string: analyzer.instace_cells.clone(),
+                iterations: 5,
+            },
+            lookup_method: LookupMethod::InlineConstraints,
+        };
+        let output_status = analyzer
+            .analyze_underconstrained(analyzer_input, &prime)
+            .unwrap()
+            .output_status;
+        assert!(output_status.eq(&AnalyzerOutputStatus::NotUnderconstrainedLocal));
+    }
```

Comment:
fn no_selector_test() {
        let circuit =
            sample_circuits::simple::no_selector::MulCircuit::<Fr>::default(
            );
        let k: u32 = 3;

        assert!(analyzer.instace_cells.clone().len().eq(&2));

        let modulus = Fr::MODULUS;
        let without_prefix = modulus.trim_start_matches("0x");
        let prime = BigInt::from_str_radix(without_prefix, 16)
            .unwrap()
            .to_string();

        let analyzer_input: analyzer_io_type::AnalyzerInput = analyzer_io_type::AnalyzerInput {
            verification_method: VerificationMethod::Random,
            verification_input: VerificationInput {
                instance_cells: HashMap::new(),
                iterations: 5,
            },
            lookup_method: LookupMethod::InlineConstraints,
        };

        let mut analyzer = Analyzer::new(
            &circuit,
            k,
            AnalyzerType::UnderconstrainedCircuit,
            Some(&analyzer_input),
        )
        .unwrap();

        let output_status = analyzer
            .analyze_underconstrained(&analyzer_input)
            .unwrap()
            .output_status;
        assert!(output_status.eq(&AnalyzerOutputStatus::NotUnderconstrainedLocal));
    }

Reply:
@sorawee Thanks for catching this. The changes look good to me. However, the test needs to be updated as we have refactored how the analyzer is initialized, which changes the tests as above.

Reply:
I'll merge the PR and will fix the test in another PR.